assets/scss/_base.scss

```diff
@@ -924,3 +933,41 @@ section#cncf {
     border-top-left-radius: 0;
   }
 }
+
+//Home page dark-mode
+@media (prefers-color-scheme: dark) {
+  .cid-home main {
+    background-color: $dark-bg-color-2;
+    color:$dark-text-color-1;
+  }
+
+  .cid-home main section:not(#video):not(#cncf) {
+    background-color: $dark-bg-color-1;
+    color:$dark-text-color-1;
+  }
+
+  .cid-home .feature-box div {
+    background-color: $dark-bg-color-2;
+    color:$dark-text-color-1;
+    h3:hover, h4:hover, h5:hover {
+      color: $dark-text-color-1;
```

Comment:
Ah, this hasn't come out how I'd imagined. Can you ensure that when hovered, these come out white?

Reply:
done